### PR TITLE
Add PR selection

### LINKED ISSUE #5: PR templates not showing up on repos?
**Describe the bug 🐞**

The PR templates are not showing up in the downstream repos. @prbzrg can you take a look to see if I did anything wrong?

### PR DESCRIPTION
**Pull request by prbzrg**
Closes #5

### CODE REVIEW

**Comment by prbzrg** [comment]
Based on:
https://github.com/kenji-miyake/test-pr-template

**Comment by prbzrg** [comment]
Let me add description too

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Click the `Preview` tab and select a PR template:
+
+- [Bug fix](?expand=1&template=bugfix.md)
```
Comment (ChrisRackauckas) [inline .github/pull_request_template.md]: Do these actually show up as links?

Comment (prbzrg) [reply .github/pull_request_template.md]: To be sure, try
https://github.com/kenji-miyake/test-pr-template/compare/main...kenji-miyake-patch-1

**Pull request merged**